Course/Course/Presentation/Details/CourseDetailsViewModel.swift

```diff
@@ -36,6 +37,11 @@ public class CourseDetailsViewModel: ObservableObject {
     let cssInjector: CSSInjector
     let connectivity: ConnectivityProtocol
     
+    var loggenInUser: Bool {
```

Comment:
@saeedbashir  did you mean 'loggedInUser' or 'loginInUser' here?

Reply:
Typo, also renamed it to `userLoggedIn` because it's returning boolean.